src/command/check-secret-expiry.ts

```diff
@@ -0,0 +1,113 @@
+import { ListAccessKeysCommand } from '@aws-sdk/client-iam';
+import chalk from 'chalk';
+
+import { AWSCredentials, ProfileConfiguration } from '../mfa-login';
+
+import { getKeyAgeInDays } from '../lib/time';
+import { createIAMClient, createStsClient } from '../lib/create-aws-client';
+
+const getKeyAgeStatus = (
+  keyAge: number,
+  keyMaxAge: number
+): { text: string; textColor: chalk.Chalk } | undefined => {
+  const daysLeft = keyMaxAge - keyAge;
+
+  if (daysLeft < 0) {
+    return { text: `✘ maximum age has exceeded`, textColor: chalk.red };
```

Comment:
Here's how I think this message should read:

> 30 days: show nothing
< 30 days: `Access key is X days old (maximum age is Y days)` (info icon/blue text)
< 7 days: `Access key is X days old (maximum age is Y days)` (warning icon/yellow text)
< 0 days: `Access key is X days old (maximum age is Y days)` (error icon/red text)